@xen-orchestra/web-core/lib/components/UiCard.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="ui-card" :style="{ flexDirection: direction, padding: padding ? '2.4rem' : 0 }">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts" setup>
+type Props = {
+  direction: 'column' | 'row'
```

Comment:
What do you think of using a prop `horizontal`? So the card layout will be vertical by default.
```suggestion
  horizontal?: boolean
```